pipedrive/types.py

```diff
@@ -84,3 +86,43 @@ def to_primitive(self, value, context=None):
             return value['id']
 
         return value
+
+
+class PipedriveListDictStringOrStringType(BaseType):
+    """
+    Pipedrive inconsistently returns values for some fields, for example: "find" for a Person just returns a string
+    for email even if there are multiple emails associated with that Person. "detail" for a Person returns a list
+    containing a dict containing strings for emails. Because of this inconsistency we need to have a field that can
+    return the correct values even if the way they are represented changes. We are opting for the list-dict-string
+    because this allows us to better control these multiple values (email addresses, phone numbers etc.).
+    """
+    def to_native(self, value, context=None):
+        if isinstance(value, list):
+            return value
+        elif isinstance(value, basestring):
```

Comment:
Does `pipedrive-py` target Python 2 exclusively, or Python 3 as well? (`basestring` doesn't exist in python 3).